syslog-ng-ctl/syslog-ng-ctl.c

```diff
@@ -191,19 +191,26 @@ slng_reload(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
 }
 
 static gboolean config_options_preprocessed = FALSE;
+static gboolean config_options_verify = FALSE;
 
 static GOptionEntry config_options[] =
 {
   { "preprocessed", 'p', 0, G_OPTION_ARG_NONE, &config_options_preprocessed, "preprocessed", NULL },
+  { "verify", 'v', 0, G_OPTION_ARG_NONE, &config_options_verify, "verify", NULL },
   { NULL,           0,   0, G_OPTION_ARG_NONE, NULL,                         NULL,           NULL }
 };
 
+
 static gint
 slng_config(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
 {
-  return _dispatch_command(config_options_preprocessed ? "CONFIG PREPROCESSED" : "CONFIG ORIGINAL");
+  if (config_options_verify)
```

Comment:
Does this new ``GET`` command has to have new argument parameter? Or should it stay the same as: `` $ syslog-ng-ctl config `` and `` $ syslog-ng-ctlconfig --preprocessed ``

Reply:
I am not 100% sure that understand you, but this should replace the current `CONFIG ORIGINAL` and `CONFIG PREPROCESSED`.

The `syslog-ng-ctl` should send these commands:
```
syslog-ng-ctl config => CONFIG GET ORIGINAL
syslog-ng-ctl config --preprocessed => CONFIG GET ORIGINAL
```

On the reciever side the 1st argument is going to be either `VERIFY` or `GET` after this, and the `ORIGINAL` and `PREPROCESSED` going to became the 2nd argument. (needs change on the reciever side also because of it)

Reply:
Yeah this was already done, I thought something needs to be done for the command line arguments passed by the user (control). But your comment confirms my understanding. Thanks!